reference/config_files/global_conf.rst

```diff
@@ -106,7 +191,47 @@ configuration as:
         name = "android_ndk"
 
         def package_info(self):
-            self.conf_info["tools.android:ndk_path"] = os.path.join(self.package_folder, "ndk")
+            self.conf_info.define("tools.android:ndk_path", os.path.join(self.package_folder, "ndk"))
 
 
 Note that this only propagates from the immediate, direct ``tool_requires`` of a recipe.
+
+
```

Comment:
maybe we could move this before the tool_requires  part and create a two sections like:
- Using config in profiles: with the content about the profiles above
- Using config in recipes, with this content